package.json

```diff
@@ -42,7 +42,7 @@
     "eslint-config-google": "^0.9.1",
     "ethereumjs-abi": "^0.6.4",
     "ethereumjs-util": "^5.1.2",
-    "ganache-cli": "^6.0.3",
+    "ganache-cli": "^6.1.0-beta.0",
```

Comment:
Updated to new beta version for ganache-cli which is supposed to have a fix for the issue of random crashes during tests which has been plaguing us for awhile